app/lib/shared/notification.dart

```diff
@@ -21,23 +21,33 @@ NotificationClient get notificationClient => ss.lookup(#_notificationClient);
 class NotificationClient {
   final http.Client _client = new http.Client();
 
-  Future notifyAnalyzer(String package, String version) =>
-      _doNotify(activeConfiguration.analyzerServicePrefix, package, version);
+  Future notifyAnalyzer(
```

Comment:
Do we need this new batching here? Notification is cheap, infrequent and can be non-blocking on the main caller. We could issue them in serial, and it would be still fine, while it would preserve the round-robin distribution of the workload.

Reply:
> Notification is cheap

Well, this can end up causing 1000 packages to be re-analyzed, so a 1000 http requests are not that cheap (and take some time as well).

Though I agree that from load balancing perspective it's not good. It did came to my mind, but then I weighted on the batch version.

Don't really have strong opinions on it.

Reply:
No strong opinions here either. Let's go with yours then.

Reply:
I've made them individual requests now, but no longer await for the notifications to be out in the upload finish callback.

(The reason it's nice to wait for things inside the request handler is that then we'll get the logging messages associated with the request. This way if the analyzer notification causes an exception we would see it. But now I trigger it in background again as before, since I call the notification many times.)